lte/gateway/python/integ_tests/s1aptests/test_multi_enb_paging_request.py

```diff
@@ -107,29 +106,20 @@ def test_multi_enb_paging_request(self):
                 req, duration=1, is_udp=True,
             ) as test:
                 response = self._s1ap_wrapper.s1_util.get_response()
-                self.assertTrue(
-                    response, s1ap_types.tfwCmd.UE_PAGING_IND.value,
-                )
+                assert response.msg_type == s1ap_types.tfwCmd.UE_PAGING_IND.value
```

Comment:
Should we use `assertEqual(...)` instead of the naked assert statement?

Reply:
The `assert` statement is the preferred method from `pytest` and `assertEqual(...)` would be the `nose`-legacy syntax here.